src/main/resources/app.conf

```diff
@@ -181,34 +196,45 @@ app {
         # the initial delay before retrying that will be multiplied with the 'factor' for each attempt
         initial-delay = ${app.sourcing.retry.initial-delay}
         initial-delay = ${?ACLS_SOURCING_RETRY_INITIAL_DELAY}
+        # the maximum delay applied to the retrying strategy
+        max-delay = ${app.sourcing.retry.max-delay}
         # maximum number of retries in case of failure (applicable only for strategy "exponential")
         max-retries = ${app.sourcing.retry.max-retries}
         max-retries = ${?ACLS_SOURCING_RETRY_MAX_RETRY}
         # the exponential factor (applicable only for strategy "exponential")
         factor = ${app.sourcing.retry.factor}
         factor = ${?ACLS_SOURCING_RETRY_FACTOR}
+        # the linear increment (applicable only for strategy "linear")
+        increment = ${app.sourcing.retry.increment}
       }
     }
 
-    # realms indexing configuration
+    # acls indexing configuration
     indexing {
       # maximum number of events taken on each batch
-      batch = ${app.indexing.batch}
-      batch = ${?ACLS_INDEXING_BATCH}
+      batch-chunk = ${app.indexing.batch-chunk}
```

Comment:
batch and chunk mean similar things; either keep the name `batch` or rename to `batch-size`